aria/orchestrator/execution_plugin/instantiation.py

```diff
@@ -110,20 +105,17 @@ def _configure_remote(operation):
 
     # Make sure we have a user
     if fabric_env.get('user') is None:
-        context = ConsumptionContext.get_thread_local()
-        context.validation.report('must configure "ssh.user" for "{0}"'
-                                  .format(operation.implementation),
-                                  level=validation.Issue.BETWEEN_TYPES)
+        reporter.report('must configure "ssh.user" for "{0}"'.format(operation.implementation),
```

Comment:
when using the consumer, the issues are appended to the context validation reports